test/controllers/content_items_controller_test.rb

```diff
@@ -137,15 +137,13 @@ class ContentItemsControllerTest < ActionController::TestCase
 
     content_store_has_item(content_item['base_path'], content_item)
 
-    with_variant EducationNavigation: "A" do
```

Comment:
In that case, you could include the `assert_response_not_modified_for_ab_test` assertion, which will check that the meta tag and response headers haven't been added.